core/base/src/TROOT.cxx

```diff
@@ -2147,9 +2147,8 @@ TClass *TROOT::LoadClass(const char *requestedname, Bool_t silent) const
 /// not it will load library "libname". If the library name does
 /// not start with "lib", "lib" will be prepended and a search will
 /// be made in the DynamicPath (see .rootrc). If not found a search
-/// will be made on libname (without "lib" prepended) and if not found
-/// a direct try of libname will be made (in case it contained an
-/// absolute path).
+/// will be made on libname (without "lib" prepended). If DynamicPathName
```

Comment:
The actual code seems to be doing the search in the reverse order.  Which is the intent?